tracpro/baseline/models.py

```diff
@@ -32,7 +32,7 @@ class BaselineTerm(models.Model):
         chained_model_field='poll',
         auto_choose=True,
         related_name="baseline_terms",
-        help_text=_("The most recent response per user will be used as the baseline.")
+        help_text=_("The least recent response per user will be used as the baseline.")
```

Comment:
Hmm, I don't think this is as clear in the reverse. What do you think about: "The first response in the time period per user..."